database/migrations/2020_03_06_000000_make_import_files_user_id_nullable.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MakeImportFilesUserIdNullable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('import_files', function (Blueprint $table) {
+            $table->string('user_id')->nullable()->change();
```

Comment:
Why do we need this to be nullable?

Reply:
When running this command from console, we want to insert a new `import_file` record, but there's no authenticated user that's uploading the file -- it's been done via the `rtv:import {reportId}` artisan command that this PR introduces.

Reply:
ah got it! thanks!